src/DependencyInjection/Configuration.php

```diff
@@ -41,6 +41,17 @@ public function getConfigTreeBuilder()
 
         $this->addBundlesSection($root);
 
+        $root
+            ->children()
+                ->arrayNode('ivory_ckeditor')
+                    ->addDefaultsIfNotSet()
+                    ->canBeEnabled()
```

Comment:
if you use this, there is no 'auto' but only true/false. see for example https://github.com/symfony-cmf/menu-bundle/blob/eb754e913e96a9fa19637bbbf86358cb86df5d54/src/DependencyInjection/Configuration.php#L63-L66